.github/actions/validate-issue/index.js

```diff
@@ -0,0 +1,121 @@
+      return;
+    }
+
+    const token = local ? process.env.GH_TOKEN : getInput("repo-token");
+
+    // Create a GitHub client
+    const octokit = getOctokit(token);
+
+    // Get owner and repo from context
+    if (local) process.env.GITHUB_REPOSITORY = "iowillhoit/gha-sandbox";
+    const owner = context.repo.owner;
+    const repo = context.repo.repo;
+    const issue_number = issue.number;
+
+    const { body } = issue;
+
+    const sfVersionRegex = "@salesforce/cli/([0-9]+.[0-9]+.[0-9]+(-[a-zA-Z0-9]+.[0-9]+)?)";
+    const sfdxVersionRegex = "sfdx-cli/([0-9]+.[0-9]+.[0-9]+(-[a-zA-Z0-9]+.[0-9]+)?)";
+    const pluginVersionsRegex = "pluginVersions|Plugin Version:";
+
+    const sfVersion = body.match(sfVersionRegex)?.[1];
+    const sfdxVersion = body.match(sfdxVersionRegex)?.[1];
+    const pluginVersionsIncluded = body.match(pluginVersionsRegex);
+
+    if ((sfVersion || sfdxVersion) && pluginVersionsIncluded) {
+      // FUTURE TODO:
+      // - Check for bundled plugins that are user installed (user) or linked (link)
+      // - Could do a check to see if the users has a prerelease version installed
+
+      if (sfVersion && sfVersion.startsWith("1.")) {
+        // TODO: Eventually suggest using sf@v2
+        const { old, latest } = compareVersions("@salesforce/cli", sfVersion);
+
+        if (old) {
+          const oldSf = getFile("./messages/old-cli.md", { THE_USER: issue.user.login, USER_CLI: "sf", USER_VERSION: sfVersion, LATEST_VERSION: latest });
+          postComment(oldSf);
+          addMoreInfoLabel();
+        }
+      }
+      if (sfdxVersion && sfdxVersion.startsWith("7.")) {
+        // TODO: Eventually suggest using sf@v2
+        const { old, latest } = compareVersions("sfdx-cli", sfdxVersion);
+
+        if (old) {
+          const oldSfdx = getFile("./messages/old-cli.md", { THE_USER: issue.user.login, USER_CLI: "sfdx", USER_VERSION: sfdxVersion, LATEST_VERSION: latest });
+          postComment(oldSfdx);
+          addMoreInfoLabel();
+        }
+      }
+
+      return;
+    } else {
+      const message = getFile("./messages/provide-version.md", { THE_USER: issue.user.login });
+      postComment(message);
+      addMoreInfoLabel();
+    }
+
+    // ---------
+    // FUNCTIONS
+    // ---------
+    async function postComment(body) {
+      const { data: comments } = await octokit.rest.issues.listComments({ owner, repo, issue_number });
+
+      // Check that this comment has not been previously commented
+      if (comments.length) {
+        if (comments.some((comment) => comment.body === body)) {
+          console.log("Already commented");
+          return;
+        }
+      }
+
+      return octokit.rest.issues.createComment({ owner, repo, issue_number, body });
+    }
+
+    function addMoreInfoLabel() {
+      return octokit.rest.issues.addLabels({
+        owner,
+        repo,
+        issue_number,
+        labels: ["more information needed"],
```

Comment:
what removes the `more information needed` label?  Should it remove `investigating`

Would it be good if additional info from the user (provided it meets the criteria) would 
1. remove `more info`
2. add `investigating` back ?

I'm trying to think of the easiest way for the Trust project to be accurate more often